addons/info.go

```diff
@@ -0,0 +1,52 @@
+package addons
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/olekukonko/tablewriter"
+	"gopkg.in/errgo.v1"
+
+	"github.com/Scalingo/cli/config"
+	"github.com/Scalingo/cli/db"
+)
+
+func Info(app, addon string) error {
+	c, err := config.ScalingoClient()
+	if err != nil {
+		return errgo.Notef(err, "fail to get Scalingo client")
+	}
+
+	addonInfo, err := c.AddonShow(app, addon)
+	if err != nil {
+		return errgo.Notef(err, "fail to get addon information")
+	}
+
+	dbInfo, err := db.Show(app, addon)
+	if err != nil {
+		return errgo.Notef(err, "fail to get database information")
+	}
+
+	forceSsl, internetAccess := "disabled", "disabled"
+	for i := range dbInfo.Features {
+		if dbInfo.Features[i]["name"] == "force-ssl" {
+			forceSsl = strings.ToLower(dbInfo.Features[i]["status"])
+		} else if dbInfo.Features[i]["name"] == "publicly-available" {
+			internetAccess = strings.ToLower(dbInfo.Features[i]["status"])
+			break
```

Comment:
You shouldn't break in this condition, there is no guarantee of the features order

```suggestion
```

Reply:
Actually there is, since you cannot make your addon publicly available if SSL isn't enforced, so it has to be after the SSL entry. I've checked. What do you think ?

Reply:
Even though you supposedly can't enforce Public availability without Force TLS, the order of the features in the array is not guaranteed at all in the code. I can imagine a scenario where this is not the case. For instance, in case of issue with these features, an operator can manually work on the database and the order of features on the database is not guaranteed

Reply:
I see, I'll remove it then. Thanks for the explanation.